realm/realm-library/src/androidTest/java/io/realm/RealmQueryTests.java

```diff
@@ -99,14 +99,14 @@ private void populateTestRealm(Realm testRealm, int dataSize) {
             allTypes.setColumnBinary(new byte[]{1, 2, 3});
             allTypes.setColumnDate(new Date(DECADE_MILLIS * (i - (dataSize / 2))));
```

Comment:
If you would be willing, I would prefer that all of those lower case 'f' float indicators were uppercased.  It isn't so important with the 'f', but getting in the habit of using upper case prevents having to distinguish between `54321` and `5432l`